### PR TITLE
DOC: Update README.md, CONTRIBUTING.md and add GOVERNANCE.md

### PR DESCRIPTION
**Pull request by ViktorvdValk**
Could everyone check if I didn't miss anyone in the GOVERNANCE.md.

And @N-Dekker could you check if the coding conventions in the CONTRIBUTING file are still correct?

The wiki is also updated now.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,12 +43,46 @@
...
+  3. Capitalize the subject line.
+  4. Do not end the subject line with a period.
+  5. Use the imperative mood in the subject line (e.g. `BUG: Fix spacing
+  not being considered.`).
+  6. Wrap the body at 80 characters.
+  7. Use semantic line feeds to separate different ideas, which improves the
+  readability.
+  8. Be concise, but honor the change: if significant alternative solutions
+  were available, explain why they were discarded.
+  9. If the commit refers to a topic discussed in [ITK's Discourse], or fixes
```
Comment (mstaring) [inline CONTRIBUTING.md:63]: ITK discourse should be updated to elastix mailing list

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,12 +43,46 @@
...
-* We will review your PR and possibly suggest changes. We also have an automatic build and test system checking all PRs ([Travis-CI](https://travis-ci.org/)). When all lights are green, the PR will be merged.
+The body of the message should clearly describe the motivation of the commit
+(**what**, **why**, and **how**). In order to ease the task of reviewing
+commits, the message body should follow the following guidelines:
+
+  1. Leave a blank line between the subject and the body.
+  This helps `git log` and `git rebase` work nicely, and allows to smooth
+  generation of release notes.
+  2. Try to keep the subject line below 72 characters, ideally 50.
+  3. Capitalize the subject line.
```
Comment (mstaring) [inline CONTRIBUTING.md:54]: that's not what we currently do right? only capitalize the specifier ENH/BUG/COMP/etc

File: README.md [hunk 3]
```diff
@@ -4,9 +4,16 @@
...
+Nowadays elastix is accompanied by [ITKElastix](https://github.com/InsightSoftwareConsortium/ITKElastix) making in available in Python ([on Pypi](https://pypi.org/project/itk-elastix/)) and by [SimpleElastix](http://simpleelastix.github.io/), making it available in languages like C++, Python, Java, R, Ruby, C# and Lua. A docker image of the latest elastix build is available as well on [dockerhub](https://hub.docker.com/repository/docker/superelastix/elastix) and several plugins exist for those who wish to use the functionality of elastix in a graphical user interface, among others a [napari](https://github.com/SuperElastix/elastix_napari) and an [3Dslicer](https://github.com/lassoan/SlicerElastix) plugin.
```
Comment (mstaring) [inline README.md:16]: making it available -> typo

Comment (mstaring) [reply README.md:16]: start a new sentence after dockerhub

**Comment by mstaring** [comment]
the wiki seems not completely working now, with Documentation leading me to an empty page

File: CONTRIBUTING.md (hunk 2 above)
Comment (ViktorvdValk) [reply CONTRIBUTING.md:54]: I interpret this as capitalize the first word of the sentence and that is what we currently do.

**Comment by ViktorvdValk** [comment]
Ah, that was a typo as well. It should be working now.

**Comment by mstaring** [comment]
lgtm, perhaps you can merge Matt's changes in here.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -43,12 +43,46 @@
...
-
-* We will review your PR and possibly suggest changes. We also have an automatic build and test system checking all PRs ([Travis-CI](https://travis-ci.org/)). When all lights are green, the PR will be merged.
+The body of the message should clearly describe the motivation of the commit
+(**what**, **why**, and **how**). In order to ease the task of reviewing
+commits, the message body should follow the following guidelines:
+
+  1. Leave a blank line between the subject and the body.
+  This helps `git log` and `git rebase` work nicely, and allows to smooth
+  generation of release notes.
+  2. Try to keep the subject line below 72 characters, ideally 50.
```
Comment (N-Dekker) [inline CONTRIBUTING.md:53]: I would prefer: Try to keep below 73 characters! (So 72 characters is still OK to me.) Problem with 73 or more is that GitHub will then "break" the subject line.

Why would 50 be the ideal? 50 characters is  usually not enough for me, when I do a commit  🤔

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -43,12 +43,46 @@
...
+  3. Capitalize the subject line.
+  4. Do not end the subject line with a period.
+  5. Use the imperative mood in the subject line (e.g. `BUG: Fix spacing
+  not being considered.`).
+  6. Wrap the body at 80 characters.
```
Comment (N-Dekker) [inline CONTRIBUTING.md:58]: I know ITK does so, placing "hard returns" (Enter) in the middle of sentences, but I very much dislike this. Modern text viewers support automatic wrapping which fit each paragraphs of a text perfectly well within the display window.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -43,12 +43,46 @@
...
+commits, the message body should follow the following guidelines:
+
+  1. Leave a blank line between the subject and the body.
+  This helps `git log` and `git rebase` work nicely, and allows to smooth
+  generation of release notes.
+  2. Try to keep the subject line below 72 characters, ideally 50.
+  3. Capitalize the subject line.
+  4. Do not end the subject line with a period.
+  5. Use the imperative mood in the subject line (e.g. `BUG: Fix spacing
+  not being considered.`).
```
Comment (N-Dekker) [inline CONTRIBUTING.md:57]: Nitpick: the example `BUG: Fix spacing not being considered.` breaks guideline 4. You may remove the period 😸  Actually I would suggest removing "not being considered" from the example, for clarity. Maybe: `BUG: Add missing spacing` ?

Comment (ViktorvdValk) [reply CONTRIBUTING.md]: Good one! It's now adjusted

**Pull request merged**